pandokia/common.py

```diff
@@ -417,11 +403,12 @@ def looks_like_a_date(name):
 
 current_user_name = None
 
-
 def current_user():
-    if 'REMOTE_USER' in os.environ:
-        return os.environ["REMOTE_USER"]
-    return 'Nobody'
+    eppn = os.getenv('HTTP_EPPN')
+    if eppn:
+        return eppn.replace('@stsci.edu', '')
```

Comment:
stsci.edu should be configured as an environment variable, rather than hardcoded.
At minimum we should file a ticket & put in a comment here with the ticket number.

Reply:
I put it in the glitch-specific configuration file, that makes more sense than setting an env variable.